components/centraldashboard-angular/frontend/src/styles.scss

```diff
@@ -7,3 +7,7 @@ body {
   margin: 0;
   font-family: Roboto, 'Helvetica Neue', sans-serif;
 }
+
+// Use the material icons offline
+$material-icons-font-path: '~material-icons/iconfont/';
```

Comment:
I am not sure we need that line, but let's see what will change when you add the common library.

Reply:
Indeed, it doesn't seem like we need it, I 'll remove it!